openslides_backend/presenter/__init__.py

```diff
@@ -6,6 +6,7 @@
     get_user_related_models,
     get_users,
     number_of_users,
+    search_users_by_name_email,
```

Comment:
Thinking about this now, I think the shorter name `search_users` is more suitable, since it describes good enough what the presenter does. What do you think?

Reply:
I think this is no `search` in a usual kind, because you need to fill the both fields in an exact manner. It's more a `get`, but not with `id`. Also we have a presenter named `get_users`.

Reply:
It is a search by name or email ;) but agree to disagree. At least I would like to rename it to `search_users_by_name_or_email` to make it sound more fluent.

Reply:
done